portals/publisher/source/src/app/components/Apis/Details/Policies/GatewaySelector.tsx

```diff
@@ -157,8 +163,8 @@ const GatewaySelector: FC<GatewaySelectorProps> = ({
                                         control={
                                             <Radio
                                                 color='primary'
-                                                disabled
-                                                onChange={handleApprovedGatewayChange}
+                                                disabled={isRestricted(['apim:api_create'], apiFromContext)}
+                                                onChange={() => handleApprovedGatewayChange(true)}
                                             />
                                         }
                                         label='Choreo Connect'
```

Comment:
Line 177: Change Typography type to h3/h4 or an appropriate one

Reply:
Fixed with: https://github.com/wso2/apim-apps/pull/211/commits/97db4049a34281272f86383e7fc0372b84f8f83c